BootloaderCommonPkg/Library/FatLib/FatLib.c

```diff
@@ -185,7 +185,7 @@ FindFile (
           }
         }
       } while (Status == EFI_SUCCESS);
-      if (EFI_ERROR (Status)) {
```

Comment:
Is this code actually a problem? It seems like coverity doesn't understand the INTN cast:
![image](https://github.com/slimbootloader/slimbootloader/assets/105564755/5b3414fa-95c6-4e29-ab5c-82cfb7b186f4)
Seems like this should be traiged as a false positive in coverity instead.

Reply:
@bejeanmo reverted changes, pls review again. thanks